.eslintrc.js

```diff
@@ -32,16 +32,9 @@ module.exports = {
       },
     ],
     'no-unused-vars': 'off',
+    'unused-imports/no-unused-imports': 'error',
     'no-undef': 'off',
     'implicit-arrow-linebreak': 'off',
-    '@typescript-eslint/no-unused-vars': [
```

Comment:
Why was this rule removed, is it to use `unused-imports/no-unused-vars`? But it seems that it has not been added.

Reply:
added at line 35, changed to `unused-imports/no-unused-vars` because `@typescript-eslint/no-unused-vars` does not provide a proper auto fix

Reply:
Line 35 is `no-unused-imports`, it should not directly include the functionality of `no-unused-vars`.

Reply:
updated